docker/k8s_tools.py

```diff
@@ -35,45 +35,70 @@ def wait_pods_running(label_selector, desired):
 
 
 def count_pods_by_phase(label_selector, phase):
-    pod_list = fetch_pods_info(label_selector)
-    filtered_pod_list = filter(lambda x: x[0] == phase, pod_list)
-    return len(filtered_pod_list)
+    pod_list = fetch_pods_info(label_selector, phase)
+    return len(pod_list)
 
 
-def fetch_pserver_ips():
-    label_selector = "paddle-job-pserver=%s" % PADDLE_JOB_NAME
-    pod_list = fetch_pods_info(label_selector)
-    pserver_ips = [item[1] for item in pod_list]
-    return ",".join(pserver_ips)
+def fetch_ips(label_selector, port=None, phase=None):
+    pod_list = fetch_pods_info(label_selector, phase)
+    ips = [item[1] for item in pod_list]
+    ips.sort()
 
+    r = []
+    for ip in ips:
+        if port != "0":
```

Comment:
Use `port != None` as it's the default value of the function arg.

Reply:
Done.